src/base/math/interpolation.h

```diff
@@ -10,7 +10,7 @@ template <typename T>
 T interpolate(const T &op0, const T &op1, double factor)
 {
 	if (factor <= 0.0) return op0;
-	else if (factor >= 1.0) return op1;
+	else if (factor >= 1.0) return T() * 0.0 + op1 * 1.0;
```

Comment:
This must be a T(), and not op0?

Reply:
This is needed because ::Anim::Interpolated contains both operand outside, and should later tell, which is which.
If we just pass op1, then Interploated will consider it as op0.
Anyway, this function should not depend on the internal working of ::Anim::Interpolated, I will solve this differently